packages/strapi-plugin-i18n/tests/locales.test.e2e.js

```diff
@@ -23,6 +23,12 @@ describe('CRUD locales', () => {
   });
 
   afterAll(async () => {
+    // set default locale back to 'en'
+    await rq({
```

Comment:
use the strapi instance to avoid using the APi outside of the test themselves when possible